pkgs/by-name/pd/pdfannots2json/package.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pdfannots2json";
+  version = "1.0.16";
+
+  src = fetchFromGitHub {
+    owner = "mgmeyers";
+    repo = pname;
```

```suggestion
    repo = "pdfannots2json;
```

See here https://github.com/nix-community/nixpkgs-lint/issues/21

---

Ok, thanks.